src/style.css

```diff
@@ -1,3 +1,70 @@
+* {
```

Comment:
Using the universal selector `*` to reset margins and paddings can impact performance. Consider using more specific selectors or a CSS reset library.